package.json

```diff
@@ -2007,7 +2007,13 @@
           "default": true,
           "description": "%cmake-tools.configuration.cmake.loadCompileCommands.description%",
           "scope": "resource"
-        }
+        },
+		"cmake.automaticReconfigure": {
+			"type": "boolean",
+			"default": true,
```

Comment:
please set the default to false

Reply:
I don’t have much insights, why do you prefer to set it to false by default?
I thought setting it to true may benefit more users than it would harm.
Also, for those that don’t want the behavior, it is more likely to actively look in the settings for a possible solution, while other users may not actively check each available configuration option and thus be unaware.

Reply:
I've been thinking about this more, and I think it would be ok for the default to be `true`.  We already reconfigure automatically when changing the variant from Debug -> Release.  Visual Studio also reconfigures automatically.

Reply:
So, I will change it back to true :)

Reply:
I just did and push a few additional changes